src/PresentationalComponents/PolicyDetailsDescription/PolicyDetailsDescription.js

```diff
@@ -30,7 +30,8 @@ const PolicyDetailsDescription = ({ policy }) => (
           <PolicyThresholdTooltip />
         </Text>
         <Text className="threshold-tooltip" component={TextVariants.p}>
-          {fixedPercentage(policy.complianceThreshold, 1)}
+          {`${fixedPercentage(policy.complianceThreshold, 1)} of rules must be
+        passed for a system to be labeled "Compliant"`}
```

Comment:
Maybe? Or do you have better inputs from PM?
```suggestion
        passed for a system to be labeled as "Compliant"`}
```

Reply:
This is the text from the mockups.

Reply:
i used the exact wording from [this mock](https://marvelapp.com/prototype/93199gg/screen/86593801)